sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -347,6 +347,72 @@ export interface KeyVaultCertificateWithPolicy extends KeyVaultCertificate {
     readonly policy?: CertificatePolicy;
 }
 
+// @public
+export const enum KnownApiVersion72Preview {
+    Seven2Preview = "7.2-preview"
+}
+
+// @public
+export const enum KnownDeletionRecoveryLevel {
+    CustomizedRecoverable = "CustomizedRecoverable",
+    CustomizedRecoverableProtectedSubscription = "CustomizedRecoverable+ProtectedSubscription",
+    CustomizedRecoverablePurgeable = "CustomizedRecoverable+Purgeable",
+    Purgeable = "Purgeable",
+    Recoverable = "Recoverable",
+    RecoverableProtectedSubscription = "Recoverable+ProtectedSubscription",
+    RecoverablePurgeable = "Recoverable+Purgeable"
+}
+
+// @public
+export const enum KnownJsonWebKeyCurveName {
+    // (undocumented)
```

Comment:
Please document these values.

Reply:
These are simple enum values that are part of the generated code which comes from the swagger. So, It should be ok even if it is not documented.